scapy/layers/tls/extensions.py

```diff
@@ -149,6 +151,30 @@ def guess_payload_class(self, p):
         return Padding
 
 
+class EncryptedServerName(Packet):
```

Comment:
Is this needed?

Reply:
Nope Just following the same format as all of the other extensions. Will remove